sdks/go/pkg/beam/core/graph/window/trigger/trigger_test.go

```diff
@@ -0,0 +1,59 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package trigger
+
+import (
+	"testing"
+	"time"
+)
+
+func TestAfterCount(t *testing.T) {
+	tr := AfterCount(1)
+	want := int32(1)
+	if got := tr.ElementCount(); got != want {
+		t.Errorf("element count not configured correctly. got %v, want %v", got, want)
+	}
+}
+
+func TestAfterProcessingTime(t *testing.T) {
```

Comment:
I think this is actually more testing the full behavior of AfterProcessingTimeTrigger, so I'd probably update to `TestAfterProcessingTimeTrigger`